provisioning/service/src/Auth/ProvisioningSasCredential.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Azure;
+using Microsoft.Azure.Devices.Common.Service.Auth;
+
+namespace Microsoft.Azure.Devices.Provisioning.Service.Auth
+{
+    internal class ProvisioningSasCredential: IAuthorizationHeaderProvider
```

Comment:
Please add meaningful doc comments for future code readers. See \iothub\service\src\DigitalTwin\Authentication\DigitalTwinSasCredential.cs for an example.